settings.gradle

```diff
@@ -18,4 +18,5 @@ include ':samples:ar-model-viewer'
 include ':samples:ar-model-viewer-compose'
 include ':samples:ar-point-cloud'
 include ':samples:model-viewer'
-include ':samples:model-viewer-compose'
\ No newline at end of file
+include ':samples:model-viewer-compose'
+include ':samples:ar-view-node'
```

Comment:
This line should be sorted with the existing ones.

Reply:
done